Resources/Prototypes/Maps/game.yml

```diff
@@ -201,3 +201,45 @@
     ChiefEngineer: [ 1, 1 ]
     ChiefMedicalOfficer: [ 1, 1 ]
     SecurityOfficer: [ 3, 6 ]
+
+- type: gameMap
+  id: boxstation
+  mapName: 'Boxstation'
+  mapNameTemplate: '{0} Boxstation {1}'
+  nameGenerator:
+    !type:NanotrasenNameGenerator
+    prefixCreator: '14'
+  mapPath: /Maps/boxstation.yml
+  minPlayers: 30
+  maxPlayers: 80
+  conditions:
+    - !type:HolidayMapCondition
+      inverted: true
+      holidays: [ "Christmas" ]
```

Comment:
no box on christmas?

Reply:
i just copied what was on the packed prototype, didnt knew it did that